v2/internal/controllers/crd_machinelearningservices_test.go

```diff
@@ -17,24 +17,25 @@ import (
 	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
 )
 
-// If recording this test, might need to manually purge the old KeyVault: az keyvault purge --name asotest-kv-qpxtvz
+// If recording this test, might need to manually purge the old KeyVault and Workspace:
+// az keyvault purge --name asotest-kv-qpxtvz
+// you need to do this for the workspace too (see https://aka.ms/wsoftdelete). As far as I can tell there's no way to do this via the az cli you have to do it via the portal.
 
 func Test_MachineLearning_Workspaces_CRUD(t *testing.T) {
 	t.Parallel()
 
 	if *isLive {
 		t.Skip("can't run in live mode, as this test is creates a KeyVault which reserves the name unless manually purged")
 	}
-
 	tc := globalTestContext.ForTest(t)
 
+	tc.AzureRegion = to.Ptr("westus3")
 	rg := tc.CreateTestResourceGroupAndWait()
 
 	sa := newStorageAccount(tc, rg)
 	kv := newVault("kv", tc, rg)
 
-	// Have to use 'eastus' location here as 'ListKeys' API is unavailable/still broken for 'westus2'
```

Comment:
Should you keep an updated variant of this comment?

Reply:
No as far as I can tell it's no longer correct.